### PR TITLE
 Initial README

### PR DESCRIPTION
**Pull request by bvallelunga**

### CODE REVIEW

File: README.md
Comment (ghost) [inline README.md]: Made an [issue](https://github.com/DopplerFoundation/dvm/issues/53) for this. I need to investigate this further.

Notes for myself: 
  * need to have brian reinstall dvm on my test droplet so I can further investigate this issue.
  * why isn't the app directory appended to the `PYTHON PATH` on the dvm side rather than the app side? 
  * why is the basedir appended to the path multiple times (ex: sentiment), cant we just do it once in `main.py` if we have to do it on the app side?

File: README.md
Comment (ghost) [inline README.md]: should be using `os.path.join`,

```python
BASE_PATH = os.path.dirname(os.path.abspath(__file__))
MODEL_PARAMS_PATH = os.join.path(BASE_PATH, 'model')
```

To understand why `os.path.join` is preferred over string concat, see [this SO post](https://stackoverflow.com/questions/13944387/why-use-os-path-join-over-string-concatenation?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa).

File: README.md
Comment (ghost) [inline README.md]: I fixed the wording:
> This document is meant to help maintainers prepare their apps for the Doppler marketplace. It includes importing guidelines as well as a list of hard dependencies and app restrictions.

File: README.md
Comment (ghost) [inline README.md]: I fixed the wording:
> All apps will be running in a [Python3](https://docs.python.org/3/) environment. The following packages will be installed by default:

File: README.md
Comment (ghost) [inline README.md]: I fixed the wording. However, this may need to change again depending on the outcome of the imports issue:
> In order for relative imports to work properly, you must:
>  1. add all files with relative imports to the `PYTHON PATH` 
>  2. remove explicit relative imports

**Review by ghost — CHANGES_REQUESTED** [review]
Minor changes for the most part. However, we need to address/further investigate the importing before merging. I need to have you setup dvm on my droplet again so I can do some follow up work on the import issue.

File: README.md
Comment (bvallelunga) [reply README.md]: dvm reinstalled. the command to reinstall
```
pip3 install git+ssh://git@github.com/DopplerFoundation/dvm.git
```

**Comment by bvallelunga** [comment]
Can you make the changes recommended in another commit. Then unblock the PR by claiming ownership.

**Review by ghost — APPROVED** [review]

**Pull request merged**
